### PR TITLE
Document needed web dependency

### PR DESCRIPTION
**Pull request by WtfJoke**
Document behaviour of #110
**What**:
That nobody with the same problem thinks its a bug.

**Why**:
By improving our documentation

**How**:

**Checklist**:

- [x] Documentation added
      
- [ ] Changelog updated
N/A
- [ ] Tests added
N/A
- [x] Ready to be merged

### CODE REVIEW

**Comment by WtfJoke** [comment]
@fletchgqc and @maiksensi can you please both verify the documented changes. 

Thanks a lot!

File: README.md [hunk 1]
```diff
@@ -174,6 +174,8 @@ Chaos Monkey for Spring Boot versions will follow the versions of Spring Boot. T
 | <b>Spring Boot 2.0.0 - 2.0.8</b>  |     <b>yes</b>     |         no         |     <b>yes</b>     |     <b>yes</b>     |     <b>yes</b>     |         <b>yes</b>          |        <b>yes</b>          |
 | <b>Spring Boot 2.1.0 - 2.1.2</b>  |         no         |         no         |         no         |         no         |     <b>yes</b>     |         <b>yes</b>          |        <b>yes</b>          |
 
+For Chaos Monkey to work, you need to have either `spring-boot-starter-web` or `spring-boot-starter-webflux` dependency in your app.
+
```
Comment (fletchgqc) [inline README.md]: I would say that "you need to have the spring-web dependency in your app. You usually get this via spring-boot-starter-web or spring-boot-starter-webflux."

Comment (WtfJoke) [reply README.md]: I rephrased the sentence like you suggested :+1:  Thanks!

**Comment by maiksensi** [comment]
LGTM!

**Pull request merged**